data/mods/Xedra_Evolved/items/inventor/artifacts.json

```
@@ -189,13 +197,17 @@
     "warmth": 20,
     "material_thickness": 3,
     "armor": [ { "encumbrance_modifiers": [ "NONE" ], "coverage": 85, "covers": [ "head" ] } ],
+    "charges_per_use": 1,
     "pocket_data": [
       {
-        "pocket_type": "MAGAZINE",
+        "pocket_type": "MAGAZINE_WELL",
+        "rigid": true,
         "airtight": true,
         "watertight": true,
-        "rigid": true,
-        "ammo_restriction": { "battery": 5000 }
+        "flag_restriction": [ "BATTERY_LIGHT", "BATTERY_MEDIUM", "BATTERY_HEAVY" ],
```

```suggestion
        "flag_restriction": [ "BATTERY_LIGHT", "BATTERY_MEDIUM" ],
```
Isn't the heavy a multi pound battery?

---

If you feel like you've balanced this around the heavy battery it'll be fine.

---

I think make the balance around middle one is okay